runtimes/onnx-runtime/stable/Dockerfile

```diff
@@ -29,7 +29,9 @@ RUN locale-gen en_US.UTF-8 && update-locale LANG=en_US.UTF-8
 # Install dependencies
 RUN pip3 install \
     onnx \
-    onnxruntime 
+    onnxruntime \
+    protobuf==3.20.2 \
+    numpy==1.22.4
```

Comment:
protobuf version is compatible with both ONNX (https://github.com/onnx/onnx/blob/rel-1.13.1/requirements.txt#L2) and ONNXRT (https://github.com/microsoft/onnxruntime/blob/main/requirements.txt.in#L5).

Same goes for numpy - ONNX (https://github.com/onnx/onnx/blob/rel-1.13.1/requirements.txt#L1) and ONNXRT (https://github.com/microsoft/onnxruntime/blob/main/requirements-dev.txt#L7).